doc/manual/src/glossary.md

```diff
@@ -73,8 +73,7 @@
 
 - [fixed-output derivation]{#gloss-fixed-output-derivation}
```

Comment:
```suggestion
- [fixed-output derivation]{#gloss-fixed-output-derivation} (FOD)
```
```suggestion
- [fixed-output derivation]{#gloss-fixed-output-derivation} (FOD, discouraged abbreviation)
```

or maybe it's ok to leave it.

Reply:
It's actually good to have, people use it a lot.